raiden/smart_contracts/ChannelManagerLibrary.sol

```diff
@@ -34,23 +34,24 @@ library ChannelManagerLibrary {
     /// @dev Get the channel of two parties
     /// @param partner (address) the address of the partner
     /// @return channel (address) the address of the NettingChannelContract of the two parties.
-    function getChannelWith(Data storage self, address partner)
+    function getChannelWith(Data storage self, address caller_address, address partner)
         constant
-        returns (address)
+        returns (address, bool, uint, uint)
     {
         uint i;
-        address[] our_channels = self.node_channels[msg.sender];
-        address channel;
+        uint j;
```

Comment:
you can initialize both `uint i;` and `uint j;` inside the `for` loop's header and save 2 lines:

`for (uint i = 0;  i < our_channels.length; ++i) {`